lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+import classnames from "classnames";
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const { innerRadiusPercent, percent, ...svgAttrs } = props;
+
+      const donutClasses = classnames('nx-binary-donut-chart', { innerRadiusPercent });
+
+      const innerRadiusPercentNotNull = innerRadiusPercent == null ? 50 : innerRadiusPercent < 0 ? 0 :
+        innerRadiusPercent > 100 ? 100 : innerRadiusPercent;
```

Comment:
I prefer to format nested ternaries as follows, so that it becomes an "if-then" on each line followed by a default value:

```suggestion
      const innerRadiusPercentNotNull = innerRadiusPercent == null ? 50 :
        innerRadiusPercent < 0 ? 0 :
        innerRadiusPercent > 100 ? 100 :
        innerRadiusPercent;
```